package.json

```diff
@@ -158,6 +158,7 @@
     "react-dnd-html5-backend": "^10.0.2",
     "react-dnd-legacy": "npm:react-dnd@^2.6.0",
     "react-dnd-touch-backend": "0.2.7",
+    "react-router-dom": "^5.2.0",
```

Comment:
I think we can remove `react-router` as well

Reply:
Very good point

Reply:
I've also accidentally added it to dependancies rather than devDependancies as well so I'll move that

Reply:
I'm also not sure if we should have `react-router-dom` as a dependency of carbon just so we can demo it in storybook but happy to be convinced otherwise